app/models/nation_inapplicability.rb

```diff
@@ -10,6 +10,7 @@ class NationInapplicability < ApplicationRecord
 
   validates :nation_id, inclusion: { in: Nation.potentially_inapplicable.map(&:id) }
   validates :alternative_url, uri: true, allow_blank: true
+  validates :alternative_url, length: { maximum: 255, too_long: "URL too long. Please shorten the URL to under 255 characters." }
```

Comment:
```suggestion
  validates :alternative_url, length: { maximum: 255, too_long: "is too long. Please shorten the URL to less than 255 characters." }
```

might be good enough for now?